### ISSUE TITLE
Reproduction of benchmark 0.2.0 results

### ISSUE THREAD

**Issue by Jsevillamol** [ctlearn-project/ctlearn#51]
I have reproduced the benchmark 0.2.0 results in the UCM server (unsure of the NVIDIA GPU model @nietootein ?).

|  Input | Telescope Type | Train Events | Val Events | Train steps | Batch size | Train time |   Val Acc  | Val Gamma Acc | Val Proton Acc |   Val AUC  |
|:------:|:--------------:|:------------:|:----------:|:-----------:|:----------:|:----------:|:----------:|:-------------:|:--------------:|:----------:|
| Single |       LST      |    161631    |    17960   |    37500    |     16     |  1h 0m 55s |  0.7034521 |   0.63914883  |    0.7676981   |  0.7905172 |
| Single |      MSTF      |    666288    |    74032   |    37500    |     16     | 1h 25m 39s |  0.7445564 |   0.8048835   |   0.68821025   |  0.8311684 |
| Single |      MSTN      |    772385    |    85821   |    37500    |     16     | 1h 29m 53s |  0.7803801 |   0.82118773  |    0.742127    |  0.8679488 |
| Single |      MSTS      |    541990    |    60222   |    37500    |     16     | 1h 14m 38s | 0.78469664 |   0.8445853   |    0.7228224   | 0.86962146 |
| Single |      SST1      |    379611    |    42180   |    37500    |     16     |   55m 59s  |  0.7793741 |   0.8133864   |   0.74446315   |  0.8592271 |
| Single |      SSTA      |    404866    |    44986   |    37500    |     16     |  1h 7m 45s | 0.72549236 |   0.66570693  |   0.78389066   |  0.8107811 |
| Single |      SSTC      |    392626    |    43626   |    37500    |     16     |  1h 5m 31s |  0.7493009 |   0.75805485  |    0.7405917   |  0.8215633 |
|  Array |       LST      |     76860    |    8541    |    37500    |     16     |   37m 27s  |  0.7280178 |   0.8020253   |    0.6643433   | 0.82055163 |
|  Array |      MSTF      |    224831    |    24982   |    37500    |     16     |  2h 7m 11s | 0.80393887 |   0.8210466   |     0.78773    |  0.895706  |
|  Array |      MSTN      |    242425    |    26937   |    37500    |     16     | 2h 11m 18s |  0.8277462 |   0.87108314  |    0.7881501   |  0.9178416 |
|  Array |      MSTS      |    200745    |    22306   |    37500    |     16     | 2h 31m 13s |  0.8198691 |   0.8415895   |    0.7982652   |  0.9099872 |
|  Array |      SST1      |    178090    |    19788   |    37500    |     16     | 3h 13m 27s |  0.7991712 |   0.7722142   |    0.8288231   | 0.89470106 |
|  Array |      SSTA      |    165302    |    18367   |    37500    |     16     |  2h 9m 23s | 0.76920563 |   0.7152889   |   0.82571363   | 0.86622685 |
|  Array |      SSTC      |    171574    |    19064   |    37500    |     16     |  1h 51m 9s |  0.8208665 |   0.81669563  |    0.8252668   | 0.90955627 |

The results are comparable to those reported in `config/v_0_2_0_benchmark/readme.md`.

The models where the difference in accuracy was more than 1% are: SSTC single telescope, SST1 array, SSTA array, SSTC array. In all those cases the difference in accuracy was below 2%.

The difference in AUROC was below 0.01 in magnitude for all models except SSTA array, where it was 0.01062685.

The difference in train times is more significant, but this can be easily explained by the use of different machines.

**Comment by aribrill** [comment]
Differences of 1% could be significant. To identify the cause, the next step should be to rerun the benchmarks with settings to eliminate any stochasticity: turn off shuffling of the TensorFlow dataset and use a specified seed to initialize the network. If significant differences are still observed, there could be a problem, otherwise it's just normal variance.
